README.md

```diff
@@ -2,42 +2,314 @@
+                "prop_missing": 0.10325476992143659,
+                "distribution": {
+                    "implements": "core.uniform_datetime",
+                    "provenance": "builtin",
+                    "class_name": "UniformDateTimeDistribution",
+                    "parameters": {
+                        "start": "2022-07-15T12:21:15",
+                        "end": "2022-08-15T10:32:05",
+                        "precision": "seconds"
+                    }
+                }
+            },
+            {
+                "name": "all_NA",
+                "type": "string",
+                "dtype": "Utf8",
+                "prop_missing": 1.0,
+                "distribution": {
+                    "implements": "core.regex",
+                    "provenance": "builtin",
+                    "class_name": "RegexDistribution",
+                    "parameters": {
+                        "re_list": [
+                            [
+                                "\\d{3,4}",
+                                0.67
+                            ]
+                        ]
+                    }
+                }
+            }
+        ]
+    }
+    
+    
+    
+    
+    
+</details> 
+
+### Generating synthetic data from a GMF file
+MetaSynth can then be used to **generate synthetic data** from any GMF standard .JSON file.
+
+![image](https://github.com/Samuwhale/metasynth/assets/22395533/37b1d7b3-7657-4478-a45d-900cce49aee5)
 
-- Automatic and manual distribution fitting
-- Generate polars DataFrame with synthetic data that resembles the original data.
-- Distributions for the most commonly used datatypes: `categorical`, `string`, `integer`, `float`, `date`, `time` and `datetime`.
-- Integrates with the [faker](https://github.com/joke2k/faker) package.
-- Structured string detection.
-- Variables that have unique values/keys.
+The generated synthetic data, known as Synthetically-Augmented Plausible dataset (as categorized by the Office for National Statistics ([ONS](https://www.ons.gov.uk/methodology/methodologicalpublications/generalmethodology/onsworkingpaperseries/onsmethodologyworkingpaperseriesnumber16syntheticdatapilot)), emulates the original data's format and plausibility at the individual record level and attempts to reproduce marginal (univariate) distributions where possible. Generated values are based on the observed distributions while adding a degree of variance and smoothing. The generated data does **not** aim to preserve the relationships between variables. The frequency of missing values and their codes are maintained in the synthetically-augmented dataset. 
 
-## Installation
+Please note: While using this type of dataset, each disclosure control must be evaluated individually, particularly in sensitive areas like names. This type of dataset is primarily used for extensive code testing, and while it has limited analytical value, there is a non-negligible risk of data disclosure.
```

Comment:
This section is based on the ONS description 

> Preserve the format and record-level plausibility as detailed previously and replicate marginal (univariate) distributions where possible.
> 
> Constructed based on the real dataset, values are generated based on observed distributions (with added fuzziness and smoothing) but no attempt made to preserve relationships.
> 
> Missing value codes and their frequency is to be preserved.
> 
> Disclosure control evaluation is necessary case by case, special care to be taken with names and so on.
> 
> To be used for extended code testing, minimal analytical value, non-negligible disclosure risk.

But I think it can be removed as it might just make the user more confused, it's something that can potentially be added and elaborated on later in the readthedocs.